4-Experimental-Accepted/RFCXXXX-Snap-Preview.md

```diff
@@ -0,0 +1,55 @@
+---
+RFC: RFC
+Author: Travis Plunk (@TravisEz13)
+Status: Experimental-Accepted
+Version: 0.1
+Area: Packaging
+Comments Due: July 15, 2018
+Plan to implement: Yes
+---
+
+# Snaps of Preview builds
+
+We currently have not published any [Snap packages](https://snapcraft.io) of a preview build.  There are some significant questions before we do.
+Primarily, do we follow Snap's model of channels or do we have a separate package called `powershell-preview`.
+I propose we have a separate package as it will allow side-by-side installation with the stable Snap and will be consistent with our Linux package names.
+
+## Motivation
+
+Snaps are easy to use and having a Snap of the preview would allow people to adopt the preview across a wide variety of platforms.
+We want to design the preview Snap in a way that allows for the preview Snap to be installed easily without losing access to the stable version of PowerShell.
+
+## Specification
+
+* There shall be two Snap packages available:
+  * Stable: a production-ready, supported build, called `powershell`.
+  * Preview: an unsupported build that may contain incomplete new features and high-impact bugs,
+    and should therefore not be used in production, called `powershell-preview`
```

Comment:
I full agree that we'll use `stable` channel as stated below and in the case we could change the sentence like
```
Preview: an unsupported build that introduce new features and high-impact bug fixes to get community feedback,  
 and should not be used in production, called `powershell-preview` .
``` 
I hope our development process ensures that every build is stable and high quality.
And please add final dot.

Reply:
This language is almost a complete cut and paste from the MSI RFC, but I agree with the change.